sherpa-onnx/python/sherpa_onnx/online_recognizer.py

```diff
@@ -46,6 +47,8 @@ def from_transducer(
         hotwords_file: str = "",
         provider: str = "cpu",
         model_type: str = "",
+        lm:str = "",
```

Comment:
```suggestion
        lm: str = "",
```

Please put a space after `:` to follow the code style.